run-tests.sh

```diff
@@ -23,10 +24,6 @@
 # waive the privileges and immunities granted to it by virtue of its status
 # as an Intergovernmental Organization or submit itself to any jurisdiction.
 
-
-pydocstyle invenio_github tests docs && \
-isort invenio_github tests --check-only --diff && \
-check-manifest --ignore ".travis-*" && \
-sphinx-build -qnNW docs docs/_build/html && \
-py.test tests && \
-sphinx-build -qnNW -b doctest docs docs/_build/doctest
+python -m check_manifest
```

Comment:
We can copy/paste our `run-tests.sh` from other modules

Reply:
I agree, to be consistent